src/tms_service.ts

```diff
@@ -110,6 +131,108 @@ export class TMSService extends AbstractEmsService {
     this._config = config;
   }
 
+  /*
+  This static function transforms a style to use the passed language
+  */
+  public static transformLanguage(
+    style: EmsVectorStyle,
+    lang: keyof typeof TMSService.SupportedLanguages
+  ): EmsVectorStyle {
+    const omtLang = TMSService.SupportedLanguages[lang]?.omtCode;
```

Comment:
If you make SupportedLanguages an array of objects we can use [`Array.prototype.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) to return the lang or `undefined` if not found.

Reply:
Done